deploy/olm-catalog/eclipse-che-preview-kubernetes/manifests/org_v1_che_crd.yaml

```diff
@@ -71,6 +71,34 @@ spec:
                     field). If omitted or left blank, it will be set to the value
                     of the `flavour` field suffixed with `-public`.
                   type: string
+                identityProviderContainerResources:
+                  description: Identity provider container custom settings
+                  properties:
+                    limits:
+                      description: Limits describes the maximum amount of compute
+                        resources allowed.
+                      properties:
+                        cpu:
+                          description: CPU, in cores. (500m = .5 cores)
```

Comment:
Here and further: please specify full form like `0.5` instead of `.5`. It is easier to read.

Reply:
It is copy/paste from go docs of the corresponding object.